### PR TITLE
playground-ios: Add account to playground app

### PR DESCRIPTION
**Pull request by tbergq**
Note that this is ok, but we should not yet tell native that
account is available.
If they try to access it in production it could crash.
We need to publish new code-push-target-version before it is safe to use
in production

### CODE REVIEW

**Comment by tbergq** [comment]
Seems I have to wait for new version of `account-native` first ⌛️

File: ios/reactNativeApp.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -468,6 +472,7 @@
 				898C440300784E69B91D0ED6 /* Roboto-Medium.ttf in Resources */,
 				21C5EFC22189F86200F3C098 /* package.json in Resources */,
 				FA705002A8FD4F5BBDB4B4F4 /* Roboto-Regular.ttf in Resources */,
```
Comment (RobinCsl) [inline ios/reactNativeApp.xcodeproj/project.pbxproj]: Should that be somehow removed?

Comment (RobinCsl) [reply ios/reactNativeApp.xcodeproj/project.pbxproj]: Actually, why is Roboto even linked for iOS? It's not used, right?

Comment (tbergq) [reply ios/reactNativeApp.xcodeproj/project.pbxproj]: I think at some historical point we probably linked it, and autolinking does not remove it. 

This does not `bubble` up to the kiwi.com app anyways.

Comment (tbergq) [reply ios/reactNativeApp.xcodeproj/project.pbxproj]: But sure, we could add an issue to remove it.

File: yarn.lock [hunk 2]
```diff
@@ -1328,20 +1328,23 @@
...
+    "@kiwicom/js" "^0.13.0"
+    "@kiwicom/orbit-components" "^0.53.0"
     "@kiwicom/orbit-design-tokens" "^0.6.5"
     "@kiwicom/relay" "^3.2.0"
     buffer "^5.2.1"
     dataloader "^1.4.0"
     date-fns "^1.30.1"
     graphql "^14.4.2"
     graphql-iso-date "^3.6.1"
+    npm-registry-client "^8.6.0"
```
Comment (RobinCsl) [inline yarn.lock]: Not related completely to this PR, but why is `npm-regristry-client` a dependency of this package?

File: yarn.lock [hunk 3]
```diff
@@ -1328,20 +1328,23 @@
...
     react-native-safe-area-view "^0.14.6"
     rimraf "^2.6.3"
```
Comment (RobinCsl) [inline yarn.lock]: Same question for `rimraf`?

File: yarn.lock (hunk 2 above)
Comment (tbergq) [reply yarn.lock]: I made a note of it, and I will improve it in the next release of account-native package.

File: yarn.lock (hunk 3 above)
Comment (tbergq) [reply yarn.lock]: Same as above

File: yarn.lock [hunk 4]
```diff
@@ -1328,20 +1328,23 @@
...
+  integrity sha512-N03UKZR6+CLQTZX9s8/JE3Six1vtxqR07TtUO4wSRxrp/Ehq84sZ3te4FF2bEdglpJnmCXfgX+jhFZXy21uNoA==
   dependencies:
     "@kiwicom/graphql-global-id" "^0.10.0"
     "@kiwicom/graphql-utils" "^0.8.0"
+    "@kiwicom/js" "^0.13.0"
+    "@kiwicom/orbit-components" "^0.53.0"
     "@kiwicom/orbit-design-tokens" "^0.6.5"
     "@kiwicom/relay" "^3.2.0"
     buffer "^5.2.1"
     dataloader "^1.4.0"
```
Comment (RobinCsl) [inline yarn.lock]: Why do you need `dataloader`? That should be useful for the GraphQL server part, not in the client, right?

Comment (tbergq) [reply yarn.lock]: Account don't have a backend yet, so it uses an in-memory-grahpql-mock, it is needed for now

Comment (tbergq) [reply yarn.lock]: Actually, I think it is in account just because someone copy-pasted it from another package. I will remove it for next release as well.

**Pull request merged**